app/src/main/res/values/strings.xml

```diff
@@ -412,26 +412,26 @@
 
     <!--Format Strings-->
     <string name="hello_client">Hello, %1$s</string>
-    <string name="double_amount">%1$.2f</string>
-    <string name="double_and_string">%1$.2f %2$s</string>
-    <string name="string_and_string">%1$s %2$s</string>
-    <string name="string_and_double">%1$s %2$.2f</string>
-    <string name="string_and_int">%1$s: %2$d</string>
+    <string name="double_amount" translatable="false">%1$.2f</string>
+    <string name="double_and_string" translatable="false">%1$.2f %2$s</string>
+    <string name="string_and_string" translatable="false">%1$s %2$s</string>
```

Comment:
For the right to left languages, the order of the two parameters has to be flipped. On the right side will be the first parameter and on the left side will be the second parameter. So I think you should keep it translatable.